pkgs/top-level/python-packages.nix

```diff
@@ -11187,19 +11227,19 @@ let
       sha256 = "16i5iksaas3m0hgbxrxpgsyri4a9ncbwbiazlhx5d6lynz1wn4m2";
     };
 
-    buildInputs = [ pkgs.gfortran ];
-    propagatedBuildInputs = with self; [ numpy ];
+    buildInputs = [ pkgs.gfortran self.nose ];
+    propagatedBuildInputs = [ self.numpy ];
 
     # TODO: add ATLAS=${pkgs.atlas}
     preConfigure = ''
-      export BLAS=${pkgs.blas} LAPACK=${pkgs.liblapack}
       sed -i '0,/from numpy.distutils.core/s//import setuptools;from numpy.distutils.core/' setup.py
     '';
 
+    preBuild = self.numpyScipyPrebuild;
```

Comment:
If `numpyScipyPrebuild` were moved into the `numpy` expression, then this expression could refer to it as `self.numpy.preBuild` instead of having to rely on the top-level attribute.